code/modules/customitems/item_spawning.dm

```diff
@@ -11,6 +11,9 @@
 		var/propadjust = query.item[2]
 		var/jobmask = query.item[3]
 		var/ok = 0
+		if(!path)
+			log_debug("Incorrect database entry found in table 'customuseritems', cuiPath is null. cuiCKey='[M.ckey]' AND (cuiRealName='[sanitizeSQL(M.real_name)]' OR cuiRealName='*'")
```

Comment:
couldn't this also be caused by the path being typed incorrectly? Or does that just do a runtime

Reply:
I'll test this case. But I think you might be right

Reply:
It also caused the same exception. Added a ispath check and the path value to the logging